docs/tethys_sdk/app_class.rst

```diff
@@ -70,14 +68,6 @@ Class Methods
 
 .. automethod:: tethys_apps.base.TethysAppBase.get_job_manager
 
-.. deprecated:: 2.1
```

Comment:
Add the docs back in for get_app_workspace and get_user_workspace but mark as deprecated in 3.